src/icon-button.jsx

```diff
@@ -28,6 +32,26 @@ const IconButton = React.createClass({
     };
   },
 
+  statics: {
+    getRelevantContextKeys(muiTheme) {
```

Comment:
I would say that this doesn't work. We need to pass `context.muiTheme` instead of `context` to the `getRelevantContextKeys`.